src/assistant/Views/AssistantDialog.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { FixedDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import { useBreakpoints } from 'cozy-ui/transpiled/react/providers/Breakpoints'
+
+import Conversation from '../Conversations/Conversation'
+import ConversationBar from '../Conversations/ConversationBar'
+import { useAssistant } from '../AssistantProvider'
+
+const AssistantDialog = () => {
+  const { assistantState, clearAssistant } = useAssistant()
+  const { isMobile } = useBreakpoints()
+  const navigate = useNavigate()
+
+  const onClose = () => {
+    navigate('..')
+    clearAssistant()
+  }
+
+  return (
+    <FixedDialog
+      open
+      fullScreen
+      size="full"
+      componentsProps={{
+        dialogTitle: { className: isMobile ? 'u-ph-0' : '' },
+        dialogActions: { className: isMobile ? 'u-mh-half' : 'u-mb-2' },
+        divider: { className: 'u-dn' }
+      }}
+      title=" "
```

Comment:
Is that unbreakable space expectd?

Reply:
oui sinon pas de "faux titre", c'est à dire d'espace blanc. Mais après vérification des maquettes, il n'y a finalement pas autant d'espace sur desktop, un space classique suffit